vms/avm/service.go

```diff
@@ -60,7 +60,10 @@ type FormattedAssetID struct {
 }
 
 // Service defines the base service for the asset vm
-type Service struct{ vm *VM }
```

Comment:
most of the code in this files has been:

- Moved to tx_builders.go file
- replaced by wallet machinery